main/SAPI.c

```diff
@@ -549,17 +549,26 @@ SAPI_API void sapi_initialize_empty_request(void)
 
 static int sapi_extract_response_code(const char *header_line)
 {
-	int code = 200;
+	zend_ulong code = 200;
 	const char *ptr;
 
 	for (ptr = header_line; *ptr; ptr++) {
 		if (*ptr == ' ' && *(ptr + 1) != ' ') {
-			code = atoi(ptr + 1);
+			const char *code_begin = ptr + 1;
+		       	char *code_end;
```

Comment:
nit: Something is wrong with the indentation here.

Reply:
Oops, I didn't catch it. I just made a fix